packages/commonwealth/client/scripts/controllers/chain/cosmos/governance-v1.ts

```diff
@@ -153,18 +144,14 @@ class CosmosGovernanceV1 extends ProposalModule<
   }
 
   private async _initProposals(proposalId?: number): Promise<void> {
-    const propToIProposal = (
-      proposal: ProposalSDKType
-    ): ICosmosProposal | null => {
-      const p = Proposal.fromSDK(proposal);
-      const status = stateEnumToString(p.status);
+    const propToIProposal = (p: ProposalSDKType): ICosmosProposal | null => {
+      const status = stateEnumToString(p.status.toString());
       const identifier = p.id.toString();
-      let title = p.title;
-      let description = p.summary;
+      let title = '';
```

Comment:
Does this mean we wont have titles for proposals anymore?

Reply:
Not top-level until Cosmos SDK 0.47. Weird thing to remove, huh?

So I'm fetching it from the first `message` in the proposals `messages`

Reply:
This sounds weird, but the default "legacy" behavior is to put the title and description into that message.

 In other words, a new proposal submission will have this behavior, until (and if) we move to V1-style proposal submission (add many messages to a single proposal, each with title, description, messageType, etc).